app/components/elements/Dialog.js

```diff
@@ -24,13 +24,21 @@ export function DialogTitle(props) {
     ...dialogTitleProps
   } = props;
 
+  const ICON_WIDTH = 28;
+
   return (
     <Flex
       p={3}
-      sx={{ alignItems: 'center', justifyContent: 'space-between', borderBottom: props.divider ? borders.divider : 'unset', ...sx }}
+      sx={{ alignItems: 'center', borderBottom: props.divider ? borders.divider : 'unset', ...sx }}
       {...dialogTitleProps}
     >
-      {children}
+      <Flex
+        // offset by Icon width to ensure Title is properly centered when closeIcon is present
+        pl={closeIcon ? ICON_WIDTH : 0}
```

Comment:
See https://github.com/tidepool-org/blip/pull/1483/files#r1873807664 for explanation

Reply:
It's tricky, but I believe the best way to do have middle piece of content be true-centered in-between elements of potentially variable sizes with flex is to use `flex-grow: 1, flex-basis: 0` on the outer elements.

I ended up having to do this on the Dialog title to accommodate adding a back button to some modals as part of the data connection work.  [Here's a link to the commit](https://github.com/tidepool-org/blip/pull/1485/commits/15bc53b554512f24df2d833541c67df34b2d67bb).